src/components/home/homeMidSection/homeMidSection.tsx

```diff
@@ -1,114 +1,229 @@
+                  </Card.Title>
+                  <Card.Text as="div">
+                    <Row className="mt-4 pt-3 fs-7 align-items-center">
+                      <Col className="text-center">
+                        <div
+                          style={{
+                            borderRadius: "50%",
+                            width: "9rem",
+                            height: "9rem",
+                            background:
+                              "linear-gradient(#e84614 5%, #CFE7CD 70%)",
+                          }}
+                          className="mx-auto"
                         >
-                          <PerfectScrollbar>
-                            <p>
-                              {x.description.split("\n").map((z, idz) => (
-                                <span
-                                  key={
-                                    "homepage_span_" +
-                                    x.name +
-                                    "_description_" +
-                                    idz
-                                  }
-                                >
-                                  {z}
-                                  <br />
-                                </span>
-                              ))}
-                            </p>
-                          </PerfectScrollbar>
-                        </Col>
-                        <Col>
-                          <img src={x.img_location} alt={x.img_alt} />
-                        </Col>
-                      </>
-                    )}
-                  </Row>
-                  <div className="text-center">
-                    {x.learn_more_href !== "" ? (
-                      <Button
-                        as="a"
-                        target="_blank"
-                        variant="white"
-                        className="shadow-md-dark text-secondary my-4 fw-bold"
-                        href={x.learn_more_href}
-                      >
-                        Learn more...
-                      </Button>
-                    ) : (
-                      <Button
-                        className="my-4 bg-primary pe-none"
-                        href={x.learn_more_href}
-                      >
-                        &nbsp;
-                      </Button>
-                    )}
-                  </div>
-                </div>
-              </Carousel.Item>
-            ))}
-        </Carousel>
-      </Col>
-    </Row>
+                          <div
+                            style={{
+                              borderRadius: "50%",
+                              width: "7rem",
+                              height: "7rem",
+                              top: "1rem",
+                              left: "1rem",
+                              position: "relative",
+                            }}
+                            className="bg-white d-flex align-items-center"
```

Comment:
In the statistics, box border should be green light

Reply:
Done